legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/runtime/connection/tempTableIsolationStrategy.pure

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import meta::relational::metamodel::*;
+import meta::relational::functions::sqlQueryToString::*;
+import meta::relational::functions::sqlQueryToString::default::*;
+
+Class <<typemodifiers.abstract>> meta::pure::alloy::connections::alloy::isolation::TempTableIsolationStrategy
+{  
+}
+
+Class meta::pure::alloy::connections::alloy::isolation::DefaultIsolationStrategy extends meta::pure::alloy::connections::alloy::isolation::TempTableIsolationStrategy
+{
+  updateNodes(u: meta::relational::commands::UpdateSQLQueryCommands[1], e: DbExtension[1], dbConfig: meta::relational::functions::sqlQueryToString::DbConfig[1])
```

Comment:
dbConfig has dbExtension and related methods, you don't need to take dbExtension as param